sources/packages/web/src/components/institutions/modals/ConfirmCOEModal.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <Dialog
+    v-if="showModal"
+    :visible="showModal"
+    :modal="true"
+    :style="{ width: '50vw' }"
+    :closable="false"
+  >
+    <template #header
+      ><p class="font-weight-bold h5">
+        Do you want to confirm this application?
+      </p></template
+    >
+    <p>
+      Confirming enrolment verifies this applicant as attending your institution
```

Comment:
please change as to is

Reply:
these contents, I copy-pasted from Figma. yes, I will update 👍